link/src/main/java/com/stripe/android/link/ui/wallet/WalletScreen.kt

```diff
@@ -208,7 +208,7 @@ internal fun WalletBody(
         Spacer(modifier = Modifier.height(12.dp))
 
         Box(modifier = Modifier.animateContentSize()) {
-            if (uiState.isExpanded) {
+            if (uiState.isExpanded || uiState.selectedItem == null) {
```

Comment:
Fixing the tests crashed on one of the test cases because of the forced unwrapping. Since this can't be statically verified, I think it's better to have the null check here instead of the force unwrap, even if it's one of those "it should never happen" cases.
The current logic is not wrong, and your suggested change seems to just invert the variables value, or am I missing something?

Reply:
My change should result in `there’s no selected item, so let’s set isExpanded to true`. However, I understand the desire for not forcefully unwrapping, so I’m fine with merging as is.

Reply:
But that's how it already works, right?

```
val isSelectedItemValid = selectedItem?.isValid ?: false

return copy(
    paymentDetailsList = response.paymentDetails,
    selectedItem = selectedItem,
    isExpanded = !isSelectedItemValid,
    isProcessing = false
)
```